container_workflow_tool/distgit.py

```diff
@@ -241,10 +256,15 @@ def dist_git_changes(self, images, rebase=False):
                 pull_upstr = image.get("pull_upstream", True)
                 repo = self._clone_downstream(component, branch)
                 df_path = os.path.join(component, "Dockerfile")
+                self.logger.info(df_path)
                 release = self._get_release(df_path)
+                downstream_from = self._get_from_df(df_path)
+                self.logger.info(f"Downstream_from: {downstream_from}\n")
                 from_tag = self.conf.get("from_tag", "latest")
                 if rebase or not pull_upstr:
-                    self.update_dockerfile(df_path, release, from_tag)
+                    self.update_dockerfile(
+                        df_path, release, self.base_image, from_tag, downstream_from=downstream_from
```

Comment:
Do we need to pass both `from_tag` and `self.base_image`? Seems like they would be used for similar use cases.

Reply:
Based on the downstream catch, It is not needed at all. Good catch.

Reply:
But I am wrong. RHEL8 needs from the tag, but RHEL7 needs self.base_image which is added into FROM.

Reply:
I have analyzed it and there is a variable `self.base_image` which can be used so, the parameter is not needed really.